Puppetfile

```diff
@@ -38,7 +38,7 @@ github "module_data", "0.0.4", :repo => "ripienaar/puppet-module-data"
 # some/most of these if you want, but it's not recommended.
 
 github "brewcask",    "0.0.7"
-github "dnsmasq",     "2.0.1"
+github "dnsmasq",     "2.0.9", :repo => "bd808/puppet-dnsmasq"
```

Comment:
we don't, I read through the commits for 2.0.9
The commits look harmless. 

If it is possible that he re-release 2.0.9 as malicious, then this is obviously an issue. We could ask this question for much of our open-sourced software though.